test/onnx/debug_embed_params.py

```diff
@@ -21,6 +21,11 @@
     print('Cannot import torch, hence caffe2-torch test will not run.')
     sys.exit(0)
 
+def check_params(onnx_model, caffe2_input):
```

Comment:
@houseroad If a pass that changes param_dict is applied, parameters list between onnx and pytorch will be different and tests in caffe2 that call debug_embed_params.py will fail. I think we should skip this test if the parameters list between onnx and pytorch is different. Do you think it's a good idea?